pyvolt-dpsim-demo/README.md

```diff
@@ -54,6 +57,9 @@ HugePages_Surp:        0
 Hugepagesize:       2048 kB
 Hugetlb:         2097152 kB
 
+If you don't see 1024 next to HugePages_Total, you may need to restart
+your system and try again with a fresh boot.
```

Comment:
I noticed on my system, that I would issue the command to set HugePages config and it wouldn't set 1024, but something less instead:
```
  lucas@debian:~/src/sogno-platform/example-deployments/pyvolt-dpsim-demo$ echo 1024 | sudo tee /proc/sys/vm/nr_hugepages
  1024
  lucas@debian:~/src/sogno-platform/example-deployments/pyvolt-dpsim-demo$ cat /proc/meminfo | grep Huge
  AnonHugePages:    430080 kB
  ShmemHugePages:        0 kB
  FileHugePages:         0 kB
  HugePages_Total:     117
  HugePages_Free:      117
  HugePages_Rsvd:        0
  HugePages_Surp:        0
  Hugepagesize:       2048 kB
  Hugetlb:          239616 kB
```
I assume this was due to my limited system resources and the programs I was running. When I restarted and issued the command on a fresh boot, it succeeded. This happened on two systems I have.

Also, should we recommend that people persist this setting or at least remind them that it's not persistent?

Reply:
The restart is a good advice. Adding a remark that this is not persistent is a good idea as well.

With the new versions of DPsim and VILLASnode we should be able to remove this requirement.